src/frequenz/sdk/timeseries/logical_meter/_formula_engine.py

```diff
@@ -35,94 +35,23 @@
 class FormulaEngine:
     """A post-fix formula engine that operates on `Sample` receivers.
 
-    Operators and metrics need to be pushed into the engine in in-fix order, and they
-    get rearranged into post-fix in the engine.  This is done using the [Shunting yard
-    algorithm](https://en.wikipedia.org/wiki/Shunting_yard_algorithm).
-
-    Example:
-        To create an engine that adds the latest entries from two receivers, the
-        following calls need to be made:
-
-        ```python
-        engine = FormulaEngine()
-        engine.push_metric("metric_1", receiver_1)
-        engine.push_oper("+")
-        engine.push_metric("metric_2", receiver_2)
-        engine.finalize()
-        ```
-
-        and then every call to `engine.apply()` would fetch a value from each receiver,
-        add the values and return the result.
+    Use the `FormulaBuilder` to create `FormulaEngine` instances.
```

Comment:
Maybe it is worth adding a cross-reference?

```suggestion
    Use the [`FormulaBuilder`][frequenz.sdk.timeseries.logical_meter.FormulaBuilder] to create `FormulaEngine` instances.
```

Reply:
But Formula* are hidden classes,  this will be a broken reference.

Reply:
Ah, OK, but at some point they will be exposed, right? Or the idea is for them to be always internal?

Reply:
I don't know of a reason to make it available to everyone yet.  So I guess it remains private until we have a reason to expose it?